python/rmm/rmm/tests/test_rmm.py

```diff
@@ -26,7 +26,7 @@
 from numba import cuda
 
 import rmm
-import rmm._cuda.stream
+import rmm.pylibrmm.stream
```

Comment:
Let's rewrite this to `from rmm.pylibrmm.stream import Stream`. Then just use `Stream` below.